### DISCUSSION TITLE
Bagaimana cara resize plot aga full/ height lebih panjang kebawah di shiny

### DISCUSSION THREAD

**Discussion by reyhanpurnawan** [teamalgoritma/community#431] [Q&A] [answered]
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/88662517/199013585-8a7982f6-1078-4cdd-908e-2aa4642ae137.png">
[Shiny CP.zip](https://github.com/teamalgoritma/community/files/9901227/Shiny.CP.zip)

**Comment by kinantytasyao** [comment, marked as answer]
Maaf bisa dilampirkan kembali zip nya? Karena zip yang dilampirkan tidak bisa dibuka

**Reply by reyhanpurnawan** [reply]
[Shiny CP.zip](https://github.com/teamalgoritma/community/files/9901699/Shiny.CP.zip)

**Reply by kinantytasyao** [reply]
Untuk plot tersebut, dapat menambahkan parameter tambahan `height` di `ggplotly` dan mengisinya sesuai dengan tinggi yang kita inginkan, sehingga kode pada line 82 di Server.R misalkan sebagai berikut:
`ggplotly(p = plot2, tooltip = "text", height = 700)`

Demikian, semoga dapat membantu.

**Reply by reyhanpurnawan** [reply]
Terima kasih
